sbt-plugin/src/main/scala/com/akkaserverless/sbt/AkkaserverlessPlugin.scala

```diff
@@ -41,14 +48,39 @@ object AkkaserverlessPlugin extends AutoPlugin {
       "Files that already exist they are not re-generated.")
     val temporaryUnmanagedDirectory = settingKey[File]("Directory to generate 'unmanaged' sources into")
     val temporaryUnmanagedTestDirectory = settingKey[File]("Directory to generate 'unmanaged' test sources into")
+    val excludeLogbackDependencies = settingKey[Boolean]("Exclude logback dependencies")
+    val excludeScalaTestDependencies = settingKey[Boolean]("Exclude scalatest dependencies")
   }
   object autoImport extends Keys
   import autoImport._
 
+  val LogbackVersion = "1.2.3"
+  val LogbackContribVersion = "0.1.5"
+  val logback = "ch.qos.logback" % "logback-classic" % LogbackVersion
+  val logbackJson = "ch.qos.logback.contrib" % "logback-json-classic" % LogbackContribVersion
+  val logbackJackson = "ch.qos.logback.contrib" % "logback-jackson" % LogbackContribVersion
+  val ScalaTestVersion = "3.2.7"
```

Comment:
I'm leaning towards not including ScalaTest dependency because we all know there are opinions about test frameworks and by having that as an explicit dependency in the example build it's easier to switch. We only depend on it in the unmanaged (one time) generated test class so users can replace that code with their own test framework code.

We have a similar approach for Java where users can switch between Junit4 and 5 or whatnot.

Reply:
ok

Reply:
removed.

Reply:
(yeah this is why I added the settings to remove the deps but yeah that was just an idea)